### DISCUSSION TITLE
Mapping mimic-iii itemid to mimic-iv

### DISCUSSION THREAD

**Discussion by box-key** [MIT-LCP/mimic-code#1172] [MIMIC-IV] [answered]
### Prerequisites

* [ X ] Put an X between the brackets on this line if you have done all of the following:
    * Checked the online documentation: https://mimic.mit.edu/
    * Checked that your issue isn't already addressed: https://github.com/MIT-LCP/mimic-code/issues?utf8=%E2%9C%93&q=

### Description

Hi, thank you very much for your amazing work. I try to map `itemids` in the `D_ITEMS` table in mimic-iii to `itemids` in the `ICU.d_items` table in mimic-iv. I found in the documentation that ICU data is coming from the Metavision ICU databases in mimic-iv, but mimic-iii also includes data from the CareVue databases, and the two databases use different itemids for the same item.

Is there any recommended way to match the `itemids` for the same item in the CareVue databases and the MetaVision databases? For example, I would like to rewrite a query like the following in mimic-iv, and what's blocking me is the mapping of itemids between two databases:
```sql
/* query for mimic-iii database */
SELECT subject_id, 
       hadm_id, 
       icustay_id, 
       itemid 
FROM   chartevents 
WHERE  itemid IN ( 6035, 3333, 938, 941,  --no match for 6035, 3333, etc in mimic-iv
                   942, 4855, 6043, 2929
                   70083, 226131, 80220 ) 
ORDER  BY subject_id, 
          hadm_id
```

Thanks

**Comment by briangow** [comment]
Since the mapping between Metavision and CareVue is not 1-to-1 this is difficult to do. More information can be found here: https://github.com/MIT-LCP/mimic-code/issues/848 . You can find common `itemids` under concepts in the code repository: https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/concepts/measurement .

**Comment by box-key** [comment, marked as answer]
Hi Brian, thank you very much for your prompt response. Yes, the concept files seem very helpful. 

For those who have similar issues, I found that some concept files under mimic-iii are useful to map `d_items.itemids` between CareVue and MetaVision. For example, https://github.com/MIT-LCP/mimic-code/blob/10613e72b2faf5a449f2a241cca1bdc86b413b05/mimic-iii/concepts/demographics/heightweight.sql#L50-L54 shows the item id for height in CareVue and Metavision. I will close this discussion now. 

Thank you for your work!
